quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -396,6 +403,10 @@ impl LossThreshold {
     }
 
     fn on_spurious_loss(&mut self, new_pkt_thresh: u64) {
+        if self.disable_dynamic_loss_threshold {
+            return;
+        }
+
```

Comment:
Would be good to have a second option to just disable this line:
`self.time_thresh = PACKET_REORDER_TIME_THRESHOLD;`

We'ld like to measure the benefits of this PR: https://github.com/cloudflare/quiche/pull/2145/files#

Reply:
Yea good point.